lib/Doctrine/DBAL/DriverManager.php

```diff
@@ -160,7 +160,30 @@ private function __construct()
      * @throws Exception
      *
      * @phpstan-param array<string,mixed> $params
-     * @psalm-param Params $params
+     * @psalm-param array{
+     *     charset?: string,
+     *     dbname?: string,
+     *     default_dbname?: string,
+     *     driver?: key-of<self::DRIVER_MAP>,
+     *     driverClass?: class-string<Driver>,
+     *     driverOptions?: array<mixed>,
+     *     host?: string,
+     *     keepSlave?: bool,
+     *     keepReplica?: bool,
+     *     master?: OverrideParams,
+     *     memory?: bool,
+     *     password?: string,
+     *     path?: string,
+     *     pdo?: \PDO,
+     *     platform?: Platforms\AbstractPlatform,
+     *     port?: int,
+     *     primary?: OverrideParams,
+     *     replica?: array<OverrideParams>,
+     *     sharding?: array<string,mixed>,
+     *     slaves?: array<OverrideParams>,
+     *     user?: string,
+     *     wrapperClass?: class-string<T>,
+     * } $params $params
```

Comment:
Duplicated `$params`

```suggestion
     * } $params
```

Reply:
af2a31e6262a4b25e61530c7ccc891f75c88fc5c